src/components/frame/v5/pages/OnboardingPage/OnboardingPage.tsx

```diff
@@ -11,7 +11,7 @@ import { LANDING_PAGE_ROUTE } from '~routes';
 import { formatText } from '~utils/intl';
 import CardConnectWallet from '~v5/shared/CardConnectWallet';
 import NotificationBanner from '~v5/shared/NotificationBanner';
-import Spinner from '~v5/shared/Spinner';
```

Comment:
Are you sure we're not using this anywhere else? I will believe you if you have checked and the answer is yes ;)

Reply:
![image](https://github.com/JoinColony/colonyCDapp/assets/23449297/1fb18319-28e2-4a8b-844a-2fd87c8b0f27)
I double checked, the `Spinner` was just wrapping `SpinnerLoader` and applying a large size for the colony preview page and user onboarding loader :+1: